src/styles/main.scss

```diff
@@ -0,0 +1,137 @@
+@import "./variables";
+@import "./reset";
+@import "./animation-stepOn60";
+@import "./extends";
+@import "./mixins";
+
+.page {
+  background: $c-bg-gradient;
+  height: 100vh;
+  margin: 0;
+
+  @extend %flex-center;
+}
+
+.watch {
+
+  & > * {
+    opacity: 0.68;
+  }
```

Comment:
Bad idea. Add opacity directly to elements or blocks.

Reply:
removed with respect to your advise